Common/Packets/Controls.cs

```diff
@@ -112,6 +132,9 @@ public Controls()
             BacktestingMaxInsights = 10000;
             MaximumDataPointsPerChartSeries = 4000;
             SecondTimeOut = 300;
+            StorageLimitMB = 5;
+            StorageFileCount = 100;
+            PersistenceIntervalSeconds = 5;
```

Comment:
Maybe `PersistenceIntervalSeconds` could be a bit bigger. It seems that we are going to be storing a significant amount of data

Reply:
If dirty; for local object store only; it would store worst-case 5mb every 5 seconds to local disk.

Reply:
oh right, I forgot about the if dirty and the 5MB limit 👍